jsonschema-core.xml

```diff
@@ -624,6 +624,18 @@
                     for registering and implementing such handlers is implementation-dependent.
                 </t>
 
+                <section title="Explicit annotation keywords" anchor="explicit-annotation">
+                    <t>
+                        The values of keywords which begin with the "@" symbol MUST be
+                        collected as annotations.
+                    </t>
+                    <t>
+                        Consequently, the "@" symbol is reserved for this purpose, and
+                        extension vocabularies which define keywords that start with "@"
+                        MUST define them as annotation-only keywords.
```

Comment:
I don't think we should allow vocabularies to define `@` keywords. `@` should explicitly mean that it's an unknown keyword. It's at best confusing to allow vocabularies to define keywords that looks exactly like non-vocabulary keywords.

Reply:
I thought it might be nice for a vocab to be able to still define a collection of them.  I don't think it hurts anything.

Reply:
Let's move this conversation back to the discussion. It seems we have more to work out.  https://github.com/orgs/json-schema-org/discussions/329#discussioncomment-5273278